### PR TITLE
add force option for mysql_db import state

### PR DESCRIPTION
**Pull request by cdalbergue**
##### ISSUE TYPE
- Feature Idea validation

##### COMPONENT NAME
mysql_db

##### ANSIBLE VERSION
```
2.4
```

##### OS / ENVIRONMENT
N/A

##### SUMMARY
When sql files are malformed, mysql_db import stops running.
mysql use the option force (-f) to avoid this problem when importing big sql files.

I think it is a good idea to integrate this option in mysql_db module and set it to false by default.

This has been tested with mysql

### CODE REVIEW

File: lib/ansible/modules/database/mysql/mysql_db.py
Comment (gundalow) [inline lib/ansible/modules/database/mysql/mysql_db.py]: `version_added: "2.4"`

Comment (cdalbergue) [reply lib/ansible/modules/database/mysql/mysql_db.py]: 👍

**Comment by gundalow** [comment]
@banyek @Jmainguy I believe you two are maintainers of MySQL modules, could you please take a look at this PR? Thanks in advance

**Review by gundalow — APPROVED** [review]

**Comment by resmo** [comment]
according the doc:
~~~
  -f, --force         Continue even if we get an SQL error. Sets
                      abort-source-on-error to 0

~~~

I am skeptical, if this option makes anything better with large files.

**Comment by bmalynovytch** [comment]
Hi @cdalbergue, thank you for your contribution.
Would you mind rebasing your branch so that it can be merged ?

**Comment by Akasurde** [comment]
@cdalbergue Are you still working on this, let us know ? Thanks, needs_info

**Comment by Akasurde** [comment]
cc @pratikgadiya12 @Andersson007

**Comment by Andersson007** [comment]
@cdalbergue at least, we need to have CI tests added to existing.
@Akasurde thanks for mentioning
I'm not sure the idea is OK or not, probably it can be useful in some cases.
I could rewrite this one after my current MySQL prs are merged.

**Comment by Andersson007** [comment]
@pratikgadiya12 feel free to rewrite this if it is ok :) be sure I'll review it if you complete this

**Comment by cdalbergue** [comment]
Hi @Andersson007

At least it has been useful for me at some point. It should be useful for other people that are in the case described.

@Akasurde I'm not working on it anymore, feel free to take the project if you want

**Comment by Andersson007** [comment]
hey dear folks! https://github.com/ansible/ansible/pull/65547
i changed and expanded it a bit, please look
thanks in advance!

**Review by bmalynovytch — APPROVED** [review]
LGTM
shipit

**Comment by cdalbergue** [comment]
Seems good to me

**Comment by bmalynovytch** [comment]
@cdalbergue could you rebase your branch on latest `devel` to fix conflicts ?

**Comment by Andersson007** [comment]
@bmalynovytch :) no needed imo :)
@cdalbergue , thanks for reviewing!

**Comment by cdalbergue** [comment]
@bmalynovytch yeah I thinks it is not needed, i think this PR could be closed when @Andersson007's is merged

**Pull request closed without merging**
